.github/workflows/molecule_tests.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Molecule Tests
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        role:
+          - deploy_user
+          # - postgresql
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install -r requirements.txt
```

Comment:
might be worthwhile at some point to add pip caching, since I think it does speed things up a bit. example from cdhweb: https://github.com/Princeton-CDH/cdh-web/blob/main/.github/workflows/unit_tests.yml#L71-L80